bgpd/bgp_updgrp_packet.c

```diff
@@ -87,39 +87,6 @@ void bpacket_queue_init(struct bpacket_queue *q)
 	TAILQ_INIT(&(q->pkts));
 }
 
-/*
- * bpacket_queue_sanity_check
- */
-void bpacket_queue_sanity_check(struct bpacket_queue __attribute__((__unused__))
```

Comment:
oh man - I don't know the specifics here, but I love a sanity check. maybe it could be under some more useful ifdef, so it's not entirely lost? or is it really adding ... nothing?

Reply:
This one didn't even compile anymore.

Reply:
I can put it back, as that I struggled with this one in all honesty.  If we are not even taking the time to make sure it works now.  What is it going to be like in another year or so?

Reply:
no, you're right - if it's not being maintained, it's not going to be "easy" to turn on and use.